docs/index.md

```diff
@@ -5624,67 +5624,87 @@ return api.combine(signedTransactions);
 
 ## submit
 
-`submit(signedTransaction: string): Promise<object>`
+`request('submit', {tx_blob: string, fail_hard: boolean}): Promise<object>`
 
-Submits a signed transaction. The transaction is not guaranteed to succeed; it must be verified with [getTransaction](#gettransaction).
+The `submit` method applies a transaction and sends it to the network to be confirmed and included in future ledgers.
```

Comment:
Docs are mostly copied from https://xrpl.org/submit.html except:
- There is no sign-and-submit mode.
- Some links were updated or removed.